src/main.rs

```diff
@@ -1,8 +1,17 @@
+extern crate env_logger;
+extern crate libc;
+#[macro_use]
+extern crate log;
+extern crate nix;
```

Comment:
nix is available for all of the major platforms, so I didn't think it was necessary to add a cfg.